### PR TITLE
Replace order_id with collection_id

### LINKED ISSUE #6: Update to support quotas download endpoint
This script downloads items from `download/:order_id/:scene_id/:asset_type`.  We should instead be downloading data with a `:collection_id` rather than an `:order_id`.

### PR DESCRIPTION
**Pull request by alukach**
This is just a simple update of the name for the id on a given download. We no longer use the concept of orders in the transition to a quotas system. 

closes #6 

This is a recreation of #12 which was merged prematurely.

### CODE REVIEW

**Comment by alukach** [comment]
I've updated this PR to allow the script to work with CSVs produced with the legacy orders-based system.

These rules are supported by the following queries on our Orders API:

```sql
orders_api_db=> SELECT left(asset_type, 6) prefix FROM assets, scenes WHERE assets.scene_id = scenes.id AND scenes.provider ILIKE 'spire' GROUP BY left(asset_type, 6);
 prefix
--------
 spire_
(1 row)

orders_api_db=> SELECT left(id, 3) prefix FROM scenes WHERE provider ILIKE 'earthdem' GROUP BY left(id, 3);
 prefix
--------
 SET
 utm
(2 rows)

**Review by Tammo-Feldmann — APPROVED** [review]
Nice foresight on making this backwards compatible with the legacy CSVs. The change makes sense to me.

**Review by edkeeble — APPROVED** [review]

File: csdap_bulk_download/cli.py [hunk 1]
```diff
@@ -91,7 +92,7 @@ def cli(
 
     \b
     The Assets CSV must contain a header row with the following columns:
-      - order_id
+      - collection_id
```
Comment (edkeeble) [inline csdap_bulk_download/cli.py:95]: The CSV search endpoint returns "provider" as the column title.

Comment (alukach) [reply csdap_bulk_download/cli.py:95]: Good catch.  We ambiguously jump between "provider" and "collection_id" throughout our codebases. At the end of the day, this actually _is_ the `collection_id`.  Should we try to adapt other parts of our system to conform to that? I feel like this ambiguity may eventually bite us.

Comment (edkeeble) [reply csdap_bulk_download/cli.py:95]: Yeah, it almost did. I'll update the stac api to return `collection_id`.

**Pull request merged**